### ISSUE TITLE
Aortic isthmus hypoplasia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7431]
**Preferred term label:**
Aortic isthmus hypoplasia
Question -- to what extent is this overlapping with [Coarctation of aorta HP:0001680](https://hpo.jax.org/app/browse/term/HP:0001680)?

AoI diameter measurement and qualitative assessment of diastolic flow have been found to be accurate indicators in the antenatal diagnosis of CoA

*The aortic isthmus (AoI) is anatomically located between the origin of the left subclavian artery and the aortic end of the ductus arteriosus (DA)

from -- https://www.karger.com/Article/Fulltext/446942

**Comment by peupeubangbang** [comment]
Hi @pnrobinson  he IPCCC suggests this would be the same as...
https://ipccc.net/ipccc-download-form/

**Hypoplastic aortic arch HP:0012304**

"Hypoplasia of the aortic arch is hypoplasia of the proximal or distal transverse arch or the aortic isthmus. The isthmus (arch between the left subclavian and insertion of the patent ductus arteriosus / ligamentum arteriosum) is hypoplastic if its diameter is less than 40% of the diameter of the ascending aorta. The proximal transverse arch (arch between the innominate and left carotid arteries) and distal transverse arch (arch between the left carotid and left subclavian arteries) are hypoplastic if their diameters are less than 60% and 50%, respectively, of the diameter of the ascending aorta."

**Comment by pnrobinson** [comment]
added this, thanks!
